src/index.ts

```diff
@@ -30,7 +30,8 @@ export type AzureDSInputOptions = {
 export class AzureDataStore {
   private path: string;
   private container: string;
-  private blobService: storage.BlobService;
+  /** Azure storage blob service instance */
+  private static blobService: storage.BlobService;
```

Comment:
What is the reason to turn this into a static member?

Reply:
moved the blobService creation outside of the class